python/paddle/v2/fluid/layers/control_flow.py

```diff
@@ -44,6 +62,48 @@ def merge_lod_tensor(in_true, in_false, x, mask, level=0):
     return out
 
 
+def Print(input,
+          first_n=-1,
+          message=None,
+          summarize=-1,
+          print_tensor_name=True,
+          print_tensor_type=True,
+          print_tensor_shape=True,
+          print_tensor_lod=True):
+    '''
+    Creates a print op that will print when a tensor is accessed.
+
+    Wraps the tensor passed in so that whenever that a tensor is accessed,
+    the message `message` is printed, along with the current value of the
+    tensor `t`.
+
+    Args:
+      input: A Tensor to print.
+      summarize: Print this number of elements in the tensor.
+      message: A string message to print as a prefix.
+      first_n: Only log `first_n` number of times.
+      print_tensor_name: Print the tensor name.
+      print_tensor_type: Print the tensor type.
+      print_tensor_shape: Print the tensor shape.
+      print_tensor_lod: Print the tensor lod.
```

Comment:
The format of doc for Python API is here: https://github.com/PaddlePaddle/Paddle/issues/6526

> Print the tensor name
whether to print ...